addon/components/frost-modal-binding.js

```diff
@@ -22,7 +32,18 @@ const FrostModalBinding = Component.extend(PropTypesMixin, {
     // Options
     animation: PropTypes.func,
     classModifier: PropTypes.string,
+    closeOnConfirm: PropTypes.bool,
     closeOnOutsideClick: PropTypes.bool,
+    // desired properties of the 'Confirm' button
+    confirm: PropTypes.shape({
+      disabled: PropTypes.bool,
+      isVisible: PropTypes.bool,
+      tabIndex: PropTypes.number,
+      text: PropTypes.string,
+      title: PropTypes.string
+    }),
+    disabledConfirmText: PropTypes.string,
+    disableConfirmUntilOnConfirmResolves: PropTypes.bool,
```

Comment:
Is this flag necessary? Can't they just not return a promise and figure out how to handle it in their own way?

Reply:
Also, if we need this - it should probably also go into the confirm namespace.

Reply:
I like your suggestion of putting the this and the alternate text in the `confirm` property.